Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2020,7 +2017,7 @@ void MediaPlayerPrivateGStreamer::handleMessage(GstMessage* message)
 
         if (GST_CLOCK_TIME_IS_VALID(gstreamerPosition))
             playbackPosition = MediaTime(gstreamerPosition, GST_SECOND);
-        if (!player->isLooping() && !eosFlagIsSetInSink && playbackPosition.isValid() && duration.isValid()
```

Comment:
if we're not checking `playbackPosition` anymore I'm not sure that local variable is worth anymore.

Reply:
indeed. The position can never be invalid. If the variable intent was only to detect when it changed from invalid to valid, we can indeed remove it.

Reply:
Actually it is used to determine if playback has finished (reached the end or the start if rate is < 0) and then log data. So I will leave it for now.